apps/civicsignalblog/src/payload/collections/Users.js

```diff
@@ -53,6 +56,76 @@ const Users = {
       },
       options: ROLE_OPTIONS,
     },
+    {
+      name: "defaultApp",
+      type: "select",
+      defaultValue: RESEARCH,
+      hasMany: false,
+      admin: {
+        isClearable: true,
+        isSortable: true,
+      },
+      options: applications,
+    },
+    {
+      name: "currentApp",
+      defaultValue: RESEARCH,
+      type: "select",
+      hasMany: false,
+      admin: {
+        isClearable: true,
+        isSortable: true,
+      },
+      options: applications,
+    },
+  ],
+  endpoints: [
+    {
+      path: "/current-app",
```

Comment:
Minor but
```suggestion
      path: "/apps/current",
```
may be a bit more RESTy? (or `users/me/apps/current`)